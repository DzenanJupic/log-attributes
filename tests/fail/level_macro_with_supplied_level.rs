#[log_attributes::info(trace, "{fn} was called")]
fn level_macro_with_supplied_level() {}

fn main() {}
