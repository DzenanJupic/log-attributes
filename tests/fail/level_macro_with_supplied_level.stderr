error: Expected a format string literal
       example: #[log(info, "{fn} returned {return}")]
 --> tests/fail/level_macro_with_supplied_level.rs:2:24
  |
2 | #[log_attributes::info(trace, "{fn} was called")]
  |                        ^^^^^
